nilrt_snac/_configs/_ssh_config.py

```diff
@@ -0,0 +1,29 @@
+import argparse
+
+from nilrt_snac._configs._base_config import _BaseConfig
+from nilrt_snac._configs._config_file import _ConfigFile
+
+from nilrt_snac import logger
+
+
+class _SshConfig(_BaseConfig):
+    def __init__(self):
+        pass  # Nothing to do for now
+
+    def configure(self, args: argparse.Namespace) -> None:
+        pass  # Nothing to do for now
+
+    def verify(self, args: argparse.Namespace) -> bool:
+        print("Verifying ssh configuration...")
+        sshd_config_file = _ConfigFile("/etc/ssh/sshd_config")
+        valid = True
+        if not sshd_config_file.exists():
+            valid = False
+            logger.error(f"MISSING: {sshd_config_file.path} not found")
+        elif not sshd_config_file.contains("ClientAliveInterval 15"):
```

Comment:
From what I read the timeout is basically interval*count in seconds, so 15*4 means 1 minute. Yes it's less than 10 minutes per spec, but this seems pretty tight to me. Do we really want it that short?

Reply:
What is the purpose of this "verify" script? If it's to make sure the target is in compliance, shouldn't it multiply the numbers and make sure they're less than 10 minutes? If a system admin changes it to 5 minutes for example, they're still in compliance, so our script failing feels wrong to me.

Reply:
There are always multiple ways to fulfil a requirement. For example, our password-quality check only verifies that libpwquality is installed, but there are other libraries that the sysadmin could use instead to enforce password quality.

We could make the check more robust like you're suggesting, but I don't think we should spend time on that unless it's a common use case that we want to explicitly support.